frontend/src/layout/elements/TextInput.js

```diff
@@ -0,0 +1,23 @@
+import Form from 'react-bootstrap/Form';
+
+export const TextInput = ({
+    inputType,
+    label,
+    onChange,
+}) => {
+
+    let textInput;
+    if (inputType) {
+        textInput = <Form.Control type={inputType} placeholder={placeholder ? placeholder : ""} onChange={onChange()} />;
+    }
+    else {
+        textInput = <Form.Control as="textarea" placeholder={placeholder ? placeholder : ""} onChange={onChange()} />;
+    }
+
+    return (
+        <Form.Group controlId={`${label.replace(/\s+/g, '')}-input-group`}>
+            <Form.Label>{label}</Form.Label>
+            {textInput}
+        </Form.Group>
```

Comment:
You didn't pass the `textInput` here. This text input won't actually have a text input.

Reply:
I did. You couldn't see it well in the code review view on Github.